### DISCUSSION TITLE
No SID in EDDB?

### DISCUSSION THREAD

**Discussion by MariusSchuller** [flybywiresim/aircraft#6384] [Help]
Fellow pilots and FBW-developers,

I wanted to fly from EDDB to LEBL today, when I discovered during cockpit-prep that I cannot select any SID from any runway in Berlin - see attached screenshots below. Is there an error or anything I can do from my side to make a departure from EDDB possible? Thank you!

Marius

![1](https://user-images.githubusercontent.com/76614781/146671776-98cdbaf7-b65f-44a1-85d4-3e1803fd75ae.PNG)
![2](https://user-images.githubusercontent.com/76614781/146671778-5f66076b-efba-4810-b9fa-d025f6942e5c.PNG)
![3](https://user-images.githubusercontent.com/76614781/146671781-2909b5ad-0509-4274-928a-2df87d9ad44b.PNG)
![4](https://user-images.githubusercontent.com/76614781/146671782-82c1b560-6e94-40b5-962a-0c2c60a0da87.PNG)

**Comment by tracernz** [comment]
Works with Navigraph 2112 for me 
![image](https://user-images.githubusercontent.com/9995998/146757223-c0b73681-35b1-48d4-9e7f-5bafce89b8f0.png)
![image](https://user-images.githubusercontent.com/9995998/146757258-5fa6c5a2-819a-4170-9d4e-ecb0c1ff4b62.png)

Do you have SIDs anywhere? Do you have SIDs in other aircraft?

**Reply by MariusSchuller** [reply]
Hi @tracernz,
yes, I do have SIDs everywhere else I have tried so far, Berlin ist the only exception. I do not have a Navigraph description.
I have not tried any other airplane so far, as I only fly the A320.

**Reply by tracernz** [reply]
Try one of the default G1000 Cessnas or something like that and see if you have SIDs for EDDB. That will determine the next troubleshooting steps.

**Reply by MariusSchuller** [reply]
Hi @tracernz ,
as you suggested, I have tried with the Cessna Citation CJ4 -> same result, no SID in EDDB but e.g. arrivals in EDDM. Seems like the root cause of this problem is outside the A32NX...

![Cessna #1](https://user-images.githubusercontent.com/76614781/147232567-aea08b9a-35cd-495e-be5f-c64d51835d6d.PNG)
![Cessna #2](https://user-images.githubusercontent.com/76614781/147232572-00f6afd2-3fda-4b91-96be-9b045cbc7553.PNG)
![Cessna #3](https://user-images.githubusercontent.com/76614781/147232573-65f885dc-219f-4d64-9360-9512d8aedb3d.PNG)
![Cessna #4](https://user-images.githubusercontent.com/76614781/147232575-843067af-42bf-4294-a4e5-6883e364792d.PNG)
![Cessna #5](https://user-images.githubusercontent.com/76614781/147232579-8ab3d4d8-6665-41ff-93cd-204fdb5634e9.PNG)

**Comment by GameKuchen** [comment]
Bump! I still have this issue

**Reply by tracernz** [reply]
Should be fixed in MSFS2024 and MSFS2020 SU16 flighting.
